### ISSUE TITLE
[Bug] Mirror error message no playing

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#4047]
### Short Description of the issue:
Previously when a brick could not be placed using the mirror tool, there was an error message alerting the player of how many blocks were not placed. This message no longer plays

### Steps to reproduce the issue:
- Create some asymmetrical geometry and enable the mirror tool
![image](https://user-images.githubusercontent.com/27857711/126037714-33e7514a-56d4-4fb9-86fb-0e8577bb1334.png)
- Place a block that when mirrored will collide and not place correctly
![image](https://user-images.githubusercontent.com/27857711/126037737-a44d3a41-9beb-4881-b83a-340e93f3271c.png)
- No error message plays, despite the mirrored block not being able to be placed

### Environment
- Starship EVO version: 21w28b
- Graphics Card: RC 580

**Comment by tsunamayo** [comment]
Yes it is because of the different framework because of prefab. I will see if I can get it back. Thanks
